<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionDialog</class>
 <widget class="QDialog" name="VersionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VersionDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFpgaVersion">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Fpga Version</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbAppVersion">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>App Version</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFpgaVersion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1.0.68</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMcuVersion">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Mcu Version</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbKernelVersion">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Kernel Version</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAppVersion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1.0.255</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMcuVersion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1.0.36</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leKernelVersion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>0.3.26.0</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
